deploy-manage/security/k8s-https-settings.md

```diff
@@ -0,0 +1,251 @@
+---
+applies_to:
+  deployment:
+    eck:
+mapped_urls:
+  - https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-tls-certificates.html
+  - https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-custom-http-certificate.html
+  - https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-kibana-http-configuration.html
+---
+
+# Manage HTTP certificates on ECK
+
+## {{es}} certificates [k8s-tls-certificates]
```

Comment:
try to avoid double headings - would be good to intro the sections we have on the page above the second heading.